README.md

```diff
@@ -2,57 +2,66 @@
 
 **This module currently only supports deploying a single task per service.**
 
-Terraform module which creates a ECS service to be deployed on the specified cluster.  
+Terraform module which creates a ECS service to be deployed Fargate or EC2.
 
 ## Examples
 
 ### Internal Service
 ```
-data "aws_iam_role" "task" {
-  name = "${var.service_name}_task"
-}
-
 module "ecs_service" {
   source                   = "git::https://github.com/bnc-projects/terraform-ecs-service.git?ref=1.0.0"
   application_path         = "/v1/service" 
   cluster_name             = "ecs-cluster-name"
-  docker_image             = "bncprojects/${var.service_name}:${var.service_version}"
   external_lb_listener_arn = ""
+  external_lb_name         = ""
   internal_lb_listener_arn = "arn:aws:elasticloadbalancing:region:account-id:listener/app/intlb"
-  java_options             = "-javaagent:newrelic/newrelic.jar -Dnewrelic.environment=${terraform.workspace} -Dnewrelic.config.file=newrelic/newrelic.yml"
+  internal_lb_name         = "intlb"
   is_exposed_externally    = false
   priority                 = 1
-  service_name             = "${var.service_name}
-  splunk_token             = "${var.splunk_token}
-  splunk_url               = "${var.splunk_url}
-  spring_profile           = "default,${terraform.workspace}"
-  task_role_arn            = "${aws_iam_role.task.arn}"
+  service_name             = var.service_name
+  task_definition_arn      = aws_ecs_task_definition.bar.arn
   vpc_id                   = "vpc-124552462"
-  tags                     = "${merge(local.common_tags, var.tags)}"
+  tags                     = merge(local.common_tags, var.tags)}
 }
 ```
 
 ### External Service
 ```
-data "aws_iam_role" "task" {
-  name = "${var.service_name}_task"
+module "ecs_service" {
+  source                   = "git::https://github.com/bnc-projects/terraform-ecs-service.git?ref=1.0.0"
+  application_path         = "/v1/service" 
+  cluster_name             = "ecs-cluster-name"
+  external_lb_listener_arn = "arn:aws:elasticloadbalancing:region:account-id:listener/app/extlb"
+  external_lb_name         = "extlb"
+  internal_lb_listener_arn = ""
+  internal_lb_name         = ""
+  is_exposed_externally    = true
+  priority                 = 5
+  service_name             = var.service_name
+  task_definition_arn      = aws_ecs_task_definition.bar.arn
+  vpc_id                   = "vpc-124552462"
```

Comment:
Not newly introduced by this PR, but should we parameterize the `vpc_id`?

Reply:
It is. 

The example usage just shows the minimal inputs required. It is up to the user of the module to provide these. How they provide them is not a concern for the module.

Reply:
Awesome - my bad! Reviewing before espresso I didn't notice the file was a `README`!